functions-and-scope.js

```diff
@@ -16,6 +16,15 @@ const grades = [9, 8, 5, 7, 7, 4, 9, 8, 8, 3, 6, 8, 5, 6];
 
 // ---- Verwachte uitkomst: 6
 
+let counter = null;
+
+for (let i = 0; i < grades.length; i++) {
+    if (grades[i] >= 8) {
+        counter += 1
```

Comment:
Dit zou je nog iets kunnen versimpelen
```suggestion
        counter++;
```